app/uk/gov/hmrc/cataloguefrontend/metrics/views/MetricsExplorerPage.scala.html

```diff
@@ -0,0 +1,314 @@
+@*
+* Copyright 2021 HM Revenue & Customs
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*@
+
+@import play.api.mvc.Call
+@import uk.gov.hmrc.cataloguefrontend.metrics.model.{Group, Repository, DependencyName, MetricsEntry}
+@import uk.gov.hmrc.cataloguefrontend.MetricsExplorerController
+@import uk.gov.hmrc.cataloguefrontend.ViewMessages
+@import uk.gov.hmrc.cataloguefrontend.{ routes => appRoutes }
+@import uk.gov.hmrc.cataloguefrontend.service.SearchByUrlService.FrontendRoutes
+@import play.twirl.api.Html
+
+@this(viewMessages: ViewMessages)
+
+@(form          : Form[_],
+groups          : Seq[Group],
+repositories    : Seq[Repository],
+dependencies    : Seq[DependencyName],
+metricsEntries  : Seq[MetricsEntry])(implicit
+request         : Request[_],
+messagesProvider: MessagesProvider
+)
+
+@standard_layout("Metrics Explorer", "search") {
+<html>
```

Comment:
Isn't standard_layout already creating the `<html><head>`?
The styles can be passed to the `head` param, but consider just adding the styles to css.

Reply:
done.